instance/models/mixins/openedx_theme.py

```diff
@@ -117,12 +127,94 @@ def get_theme_settings(self):
             "EDXAPP_DEFAULT_SITE_THEME": "simple-theme",
             "SIMPLETHEME_EXTRA_SASS": """
                 .global-header {{
-                    background: {header_bg};
+                    background: {header_bg}; // Header color
+
+                    span {{
+                      color: {link_color}; // Link Color
+                    }}
+
+                    .nav-links .secondary {{
+                      a.sign-in-btn,
+                      a.register-btn:hover
+                      {{
+                        color: {header_bg} !important; // Header color
```

Comment:
@giovannicimolin, did you check if you could add more specific selectors instead of using `!important` which is generally not a good thing to do?